platform/broadcom/sonic-platform-modules-inventec/d7264q28b/modules/gpio-ich.c

```diff
@@ -0,0 +1,513 @@
+/*
+ * Intel ICH6-10, Series 5 and 6, Atom C2000 (Avoton/Rangeley) GPIO driver
```

Comment:
this file is already in linux kernel, why not use that?

Reply:
Because we cannot find the ko file. 

root@SONiC-Inventec-d7032-100:/lib/modules/4.9.0-7-amd64# find . -name "gpio*"
./kernel/drivers/gpio
./kernel/drivers/gpio/gpio-amdpt.ko
./kernel/drivers/gpio/gpio-generic.ko
./kernel/drivers/gpio/gpio-kempld.ko
./kernel/drivers/gpio/gpio-ml-ioh.ko
./kernel/drivers/gpio/gpio-pca953x.ko
./kernel/drivers/gpio/gpio-sch.ko
./kernel/drivers/gpio/gpio-viperboard.ko
./kernel/drivers/input/keyboard/gpio_keys.ko

Reply:
please enable CONFIG_GPIO_ICH in sonic linux kernel

Reply:
Done. (https://github.com/Azure/sonic-linux-kernel/pull/59)